pkg/cpi/mockingUtils.go

```diff
@@ -433,9 +471,119 @@ func GetFunctionNameByTestTypeAndMethod(method, testType string) string {
 		if method == "POST" {
 			return "UpdateIntegrationDesigntimeArtifactNegative"
 		}
+
+	case "PostiveAndDeployIntegrationDesigntimeArtifactResBody":
+		if method == "POST" {
+			return "IntegrationArtifactDeploy"
+		}
+
+	case "NegativeAndDeployIntegrationDesigntimeArtifactResBody":
```

Comment:
How about adding this case to the above case with a comma seperator, as in:
```suggestion
	case "PostiveAndDeployIntegrationDesigntimeArtifactResBody",
		"NegativeAndDeployIntegrationDesigntimeArtifactResBody":
```

Reply:
since PostiveAndDeployIntegrationDesigntimeArtifactResBody and NegativeAndDeployIntegrationDesigntimeArtifactResBody call different functions for returning response body. both cant be combined as same switch case.

Reply:
I don't understand why not, to me both cases look exactly the same. 

Here is the `PostiveAndDeployIntegrationDesigntimeArtifactResBody` case:
```
		if method == "POST" {
			return "IntegrationArtifactDeploy"
		}
```

Here is the `NegativeAndDeployIntegrationDesigntimeArtifactResBody` case:
```
		if method == "POST" {
			return "IntegrationArtifactDeploy"
		}
```

Where is the difference?

Reply:
fixed in latest commit..